server/routes/journeys/record-investigation/evidence-secured/view.njk

```diff
@@ -1,30 +1,36 @@
 {% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
 {% from "govuk/components/details/macro.njk" import govukDetails %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "partials/submitButton/macro.njk" import submitButton %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 
 {% extends "partials/layout.njk" %}
 
-{% set pageTitle = "Evidence secured - Record a CSIP investigation" %}
+{% set pageTitle = "Evidence secured - " + ('Update' if isUpdate else 'Record') + " a CSIP investigation" %}
 
 {% block innerContent %}
-    <span class="govuk-caption-l">Record a CSIP investigation</span>
+    <span class="govuk-caption-l">{{'Update' if isUpdate else 'Record'}} a CSIP investigation</span>
```

Comment:
there's a `{% include 'base/components/investigation-caption.njk' %}` for this caption